storybooknative/src/Button/index.js.flow

```diff
@@ -0,0 +1,32 @@
+// @flow
+
+import { Text } from "react-native";
+import * as React from "react";
+
+import { TOKENS } from "./consts";
+
+export type Type =
+  | "primary"
+  | "secondary"
+  | "info"
+  | "success"
+  | "warning"
+  | "critical"
+  | "facebook"
+  | "google";
+
+export type Size = "small" | "normal" | "large";
+
+export type Tokens = $Keys<typeof TOKENS>;
+
+export type Props = {|
+  +children?: React.Element<Text> | string,
```

Comment:
What if we pass it our Translation component?

Reply:
So it probably should be `React.Node` to be more permissive. Or can't we create some `abstract` type, like `Text`, and then leave the implementation/description of this type to the people using the library?

Reply:
How would you do it syntactically?

Reply:
Why limit it to text, I could pass a new inside a button as well. For me, React.Node is fine.